client/my-sites/domains/domain-management/edit/domain-types/mapped-domain-type.jsx

```diff
@@ -70,16 +77,20 @@ class MappedDomainType extends React.Component {
 				}
 			);
 		} else {
+			setupInstructionsMessage = translate(
+				'Follow these instructions to set up your domain mapping:'
```

Comment:
How about something even more explicit (that tells the user also _why_ they need to finish this process - they might think it's optional or something). This pulls in from the "recommended setup" step, so that it's more visible:

> You need to follow these instructions to finish connecting the {domainName} domain to your WordPress.com site:

"Follow these instructions to set up your domain mapping" might just not be enough to give the user the hint that this is what they need to do in order to make their site work as expected.